Libraries/Image/RCTImageLoader.m

```diff
@@ -377,10 +371,24 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
                                                    userInfo:nil], nil);
           return;
         }
-      }
 
-      // Call handler
-      completionHandler(nil, data);
+        NSString *responseDate = ((NSHTTPURLResponse *)response).allHeaderFields[@"Date"];
```

Comment:
I just amend commit and pushed, so this comment has outdated.
So I'll mention you at about line 380 again. See you at there.